exc3/KhoiB.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace exc3
+{
+    internal class KhoiB : ThiSinh
+    {
+        public const string? TOAN = "TOAN";
```

Comment:
why do you set this as nullable?

Reply:
Sorry about that , this my bad , i will fix it now .